### PR TITLE
Small analytics additions

### PR DESCRIPTION
**Pull request by ebrodymoore**
Added data pipelines, tools outside of Looker and Prometheus search data

### CODE REVIEW

File: handbook/bizops/index.md [hunk 1]
```diff
@@ -29,12 +29,20 @@ Here are the following sources we collect data from:
 * Sourcegraph.com Site-admin pages: customer subscriptions and license keys
 * [Pings](https://docs.sourcegraph.com/admin/pings) from self-hosted Sourcegraph instances containing anonymous and aggregated information
 * [Custom tool to track events](https://github.com/sourcegraph/sourcegraph/issues/5486) on the Sourcegraph.com instance
+* [Prometheus search data](https://sourcegraph.com/-/debug/grafana/d/0ZBSEmRWk/searcher?orgId=1&refresh=5s) represents the breakdown of search statuses (success, error, timeout, partial timeout) on [Sourcegraph.com](https://sourcegraph.com/search)
```
Comment (dadlerj) [inline handbook/bizops/index.md]: How did you pick this one? If I'm not mistaken, there are a number of dashboards — if you haven't already, Uwe or someone else on the distribution team may be able to help find useful ones.

File: handbook/bizops/index.md [hunk 2]
```diff
@@ -29,12 +29,20 @@ Here are the following sources we collect data from:
...
 
 ### Data tools
 
 * [Looker](#using-looker): Business intelligence/data visualization tool
 * Google Cloud Platform: BigQuery is our data warehouse and the database Looker runs on top of
 * Google Sheets: There are a [number of spreadsheets](https://drive.google.com/drive/folders/1vOyhFO90FjHe-bwnHOZeljHLuhXL2BAv)that Looker queries (by way of BigQuery).
+* BizOps does built ah-hoc tools to analyze data for various reasons. Currently, the only major tool [analyzes Sourcegraph.com search data](https://docs.google.com/spreadsheets/d/1C7ZMcurYv48CmKr_B1_rFNBdFazsxNq5c-wZMlUQjXQ/edit#gid=1679017494). All scripts are in the [analytics repo](https://github.com/sourcegraph/analytics).
```
Comment (dadlerj) [inline handbook/bizops/index.md]:
```suggestion
* BizOps does build ad-hoc tools to analyze data for various reasons. Currently, the only major tool [analyzes Sourcegraph.com search data](https://docs.google.com/spreadsheets/d/1C7ZMcurYv48CmKr_B1_rFNBdFazsxNq5c-wZMlUQjXQ/edit#gid=1679017494). All scripts are in the [analytics repo](https://github.com/sourcegraph/analytics). 
```

Comment (dadlerj) [reply handbook/bizops/index.md]: This is an interesting section... I'm torn between trying to list all scratch analyses we ever make like this one as sub-bullets here, or going the opposite direction and instead not trying to be comprehensive here at all (and instead saying "just check the analytics repo"). What's your take?

File: handbook/bizops/index.md [hunk 3]
```diff
@@ -29,12 +29,20 @@ Here are the following sources we collect data from:
...
 * Google Cloud Platform: BigQuery is our data warehouse and the database Looker runs on top of
 * Google Sheets: There are a [number of spreadsheets](https://drive.google.com/drive/folders/1vOyhFO90FjHe-bwnHOZeljHLuhXL2BAv)that Looker queries (by way of BigQuery).
+* BizOps does built ah-hoc tools to analyze data for various reasons. Currently, the only major tool [analyzes Sourcegraph.com search data](https://docs.google.com/spreadsheets/d/1C7ZMcurYv48CmKr_B1_rFNBdFazsxNq5c-wZMlUQjXQ/edit#gid=1679017494). All scripts are in the [analytics repo](https://github.com/sourcegraph/analytics). 
+
+### Data pipelines
+
+Every data source is assumed to always be up-to-date unless noted otherwise.  
+
+* HubSpot data: Every week on Sunday night or Monday morning, BizOps triggers [scripts](https://github.com/sourcegraph/analytics/tree/master/HubSpot%20ETL) that retrieve data from HubSpot and upload it to our database. This does mean that currently our HubSpot data in Looker is not updated during the week. If you have a particular reason you'd like it to be updated during the week, mention @ericbm in the #analytics channel in Slack.
```
Comment (dadlerj) [inline handbook/bizops/index.md]:
```suggestion
* HubSpot data: Every week on Sunday night or Monday morning, BizOps triggers [scripts](https://github.com/sourcegraph/analytics/tree/master/HubSpot%20ETL) that retrieve data from HubSpot and upload it to BigQuery. This does mean that currently our HubSpot data in Looker is not updated during the week. If you have a particular reason you'd like it to be updated during the week, mention @ericbm in the #analytics channel in Slack. 
```

File: handbook/bizops/index.md (hunk 2 above)
Comment (ebrodymoore) [reply handbook/bizops/index.md]: Actually, just thought of a better solution (I think). I'll link it to a Google Drive folder where we have outputs from tools and also link to the repo for the source code. Thoughts?

File: handbook/bizops/index.md (hunk 1 above)
Comment (ebrodymoore) [reply handbook/bizops/index.md]: Good call, that was the one we were specifically using for an OKR but I'll share the folder of dashboards instead

**Comment by ebrodymoore** [comment]
> Looks great!
> 
> RE: "every data source is assumed to be up-to-date" I should probably go and delete all of the old looker charts 😂

@dadlerj haha I actually meant the underlying data so I'll make that change, but that too! I did archive a lot last month so we're not in too bad of shape here 😄

**Pull request merged**